mmv1/products/integrationconnectors/Connection.yaml

```diff
@@ -42,61 +42,68 @@ examples:
     primary_resource_id: "pubsubconnection"
     vars:
       connection_name: "test-pubsub"
+    skip_test: true
     ignore_read_extra:
-      - 'status.0.description'
+      - "status.0.description"
```

Comment:
Could you revert all these quote mark changes?

Reply:
Done